autoPyTorch/datasets/base_dataset.py

```diff
@@ -24,7 +24,8 @@
     HoldoutValTypes,
     NoResamplingFunc,
     NoResamplingFuncs,
-    NoResamplingStrategyTypes
+    NoResamplingStrategyTypes,
+    RESAMPLING_STRATEGIES
```

Comment:
I do not know why MyPy does not raise error, but `[CrossValTypes, HoldoutValTypes]` should be `Union[CrossValTypes, HoldoutValTypes]`